spring-rabbit/src/main/java/org/springframework/amqp/rabbit/connection/SimpleConnection.java

```diff
@@ -124,6 +124,10 @@ public int getPort() {
 		return this.delegate.getPort();
 	}
 
+	com.rabbitmq.client.Connection getDelegate() {
```

Comment:
Yes, it was intended; I don't think we need to allow end-users to get to the delegate.